app/controllers/trainees_controller.rb

```diff
@@ -12,8 +12,12 @@ def new
   end
 
   def create
-    trainee = Trainee.create!(trainee_params)
-    redirect_to trainee_path(trainee)
+    if trainee_params[:record_type] == "other"
```

Comment:
This could be a guard clause, `other_route?` being a private method to check the parameters.

```suggestion
   redirect_to trainees_not_supported_route_path and return if other_route?
```

Reply:
I think we will leave this as is for now because its likely to change when we start adding validation to the pages.